frontend/src/metabase/ui/components/icons/Icon/icons/warning_triangle_filled.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M9.2774 2.12205C8.69167 1.17025 7.30815 1.17025 6.72243 2.12205L0.51199 12.214C-0.103029 13.2134 0.615996 14.5002 1.78948 14.5002H14.2103C15.3838 14.5002 16.1029 13.2134 15.4878 12.214L9.2774 2.12205ZM7.2501 8.50016C7.2501 8.91437 7.58589 9.25016 8.0001 9.25016C8.41431 9.25016 8.7501 8.91437 8.7501 8.50016V6.00016C8.7501 5.58594 8.41431 5.25016 8.0001 5.25016C7.58589 5.25016 7.2501 5.58594 7.2501 6.00016V8.50016ZM7.0001 11.5002C7.0001 10.9479 7.44782 10.5002 8.0001 10.5002C8.55239 10.5002 9.0001 10.9479 9.0001 11.5002C9.0001 12.0525 8.55239 12.5002 8.0001 12.5002C7.44782 12.5002 7.0001 12.0525 7.0001 11.5002Z" fill="#E8B923" />
```

Comment:
Icons usually have `fill="currentColor"` so they can be styled.

Reply:
☝️

Reply:
Experts agree.
I had to add `iconColor` prop to undos: 91a80658ac30c66072d962f839733cb88d875a1e